specification/containerservice/resource-manager/Microsoft.ContainerService/fleet/readme.md

```diff
@@ -38,22 +38,22 @@ input-file:
   - stable/2024-04-01/fleets.json
 ```
 
-### Tag: package-2023-10
```

Comment:
TypeSpec orders versions by the way we put it in main.tsp. If grouping GA APIs together, 2023-10 will be considered as later than 2024-02-preview. The order doesn't matter much in readme files perhaps, but it is good to keep consistent.